src/edu/stanford/rsl/tutorial/fourierConsistency/coneBeam/MovementCorrection3D.java

```diff
@@ -644,7 +780,7 @@ public double evaluate(double[] x, int block){
 			//time = time1;
 			//CONRAD.log("Time for 2) Setting the motion vector:  " + diff/1e6);
 
-			applyShift();
+			applyShift(); //shifts are applied to m_2dFourierTransposed, result is written in m_3dFourierTransposed
```

Comment:
is that comment correct? I thought it is done in-place in m_2dFourierTransposed? Can you please double check?